src/App.jsx

```diff
@@ -0,0 +1,40 @@
+
+import React, { Component } from 'react';
+import $ from 'jquery';
+import WelcomeModal from './WelcomeModal';
+import Menu from './Blog';
```

Comment:
```suggestion
import Blog from './Blog';
```
скорее `Blog`, чем `Menu` :)